src/features/common/Layout/Navbar/index.tsx

```diff
@@ -1,19 +1,10 @@
-import Link from 'next/link';
-import { useRouter } from 'next/router';
 import React, { useContext, useEffect, useState } from 'react';
-import { useLocale, useTranslations } from 'next-intl';
-import Me from '../../../../../public/assets/images/navigation/Me';
-import MeSelected from '../../../../../public/assets/images/navigation/MeSelected';
-import { ThemeContext } from '../../../../theme/themeContext';
-import themeProperties from '../../../../theme/themeProperties';
-import getImageUrl from '../../../../utils/getImageURL';
 import { useUserProps } from '../UserPropsContext';
-import GetNavBarIcon from './getNavBarIcon';
-import GetSubMenu from './getSubMenu';
```

Comment:
If we simplify the navigation this way it would be nice to clean up the codebase from all unused components, assets, resources and more to not leave around too much dead code making any future development more difficult.